backend/src/main/java/backend/backend/exception/globalcontrolleradvice/ControllerAdvice.java

```diff
@@ -24,9 +24,16 @@ public ExceptionMessage notFoundException(NotFoundException e) {
         return new ExceptionMessage(e.getMessage());
     }
 
+    @ExceptionHandler(InvalidVerificationCodeException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public ExceptionMessage invalidVerificationCodeException(InvalidVerificationCodeException e) {
```

Comment:
```suggestion
    public ExceptionMessage invalidVerificationCodeException(BussinessException e) {
```

BussinessException를 상속 받았으니 위와 같이 예외처리 하면 BussinessException를 상속받은 모든 예외를 한번에 처리할 수 있어요!

[참고](https://github.com/kim1387/travel/blob/0644702073d50e7f19bbaddbf1390890b43c89c9/src/main/java/com/kim1387/travel/global/exception/handler/GlobalExceptionHandler.java#L18)

Reply:
아 그럼 InvalidVerificationCodeException이것과 같이 BussinessException을 상속받은 예외들은 해당 예외발생만 시키는 용도로 사용하고  실제 예외처리는 상위 클래스인 BussinessException으로 실행하도록 하는게 좋다는 말씀이신거죠??

InvalidVerificationCodeException 발생  -> InvalidVerificationCodeException ->  BussinessException -> RunTimeException 이 아닌

InvalidVerificationCodeException 발생 ->  BussinessException -> RunTimeException 인 방식이 맞을까요?

Reply:
네 맞습니다! 사실 BussinessException 사용하는 이유기도 합니다. Error를 한곳에서 핸들링 하다보니
에러를 관리하는 전략을 한번에 통일하여 관리할 수도 있고, GlobalExceptionHandler 클래스가 너무 길어지지 않도록 할 수 있습니다